lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -94,12 +94,36 @@ SPIRVExtInst *SPIRVToLLVMDbgTran::getDbgInst(const SPIRVId Id) {
         EI->getExtSetKind() == SPIRV::SPIRVEIS_OpenCL_DebugInfo_100 ||
         EI->getExtSetKind() ==
             SPIRV::SPIRVEIS_NonSemantic_Shader_DebugInfo_100 ||
-        EI->getExtSetKind() == SPIRV::SPIRVEIS_NonSemantic_Kernel_DebugInfo_100)
+        EI->getExtSetKind() == SPIRV::SPIRVEIS_NonSemantic_Shader_DebugInfo_200)
       return EI;
   }
   return nullptr;
 }
 
+// Check if module is generated using NonSemantic.Shader.DebugInfo.XXX
+// extended instruction set
+inline bool isNonSemanticDebugInfo(const SPIRVExtInstSetKind Kind) {
+  return (Kind == SPIRVEIS_NonSemantic_Shader_DebugInfo_100 ||
+          Kind == SPIRVEIS_NonSemantic_Shader_DebugInfo_200);
+}
+
+// Get integer parameter of debug instruction considering whether it's
+// Literal or <id> of OpConstant instruction depending on DebugInfo
+// iextended instruction set kind
+SPIRVWord SPIRVToLLVMDbgTran::getConstantOrLiteral(
```

Comment:
```suggestion
SPIRVWord SPIRVToLLVMDbgTran::getConstantValueOrLiteral(
```
Just to avoid the confusion. At first glance I though this could return either Constant Id or Literal.